libs/ui/src/lib/treemap-chart/treemap-chart.component.ts

```diff
@@ -86,7 +86,32 @@ export class GfTreemapChartComponent
     this.isLoading = true;
 
     const { endDate, startDate } = getIntervalFromDateRange(this.dateRange);
+    const netPerformancePercentsWithCurrencyEffect = this.holdings.map((h) =>
+      getAnnualizedPerformancePercent({
+        daysInMarket: differenceInDays(
+          endDate,
+          max([h.dateOfFirstActivity ?? new Date(0), startDate])
+        ),
+        netPerformancePercentage: new Big(
+          h.netPerformancePercentWithCurrencyEffect
+        )
+      }).toNumber()
+    );
 
+    const positiveNetPercents = netPerformancePercentsWithCurrencyEffect.filter(
+      (v) => v > 0
+    );
+    const positiveNetPercentsRange = {
```

Comment:
Add a line break and refactor to

`const positiveNetPerformancePercentsRange = ...`